### PR TITLE
Cursor and star tweaks

### PR DESCRIPTION
**Pull request by owennw**
Response to the second point of #729

### CODE REVIEW

**Comment by WPFerg** [comment]
LGTM. One question: On the card tearout, should the cursor revert back to default, or should it still be `pointer`?

**Comment by owennw** [comment]
¯\_(ツ)_/¯

@rmarcalo ?

File: src/sidebars/star/star-controller.js [hunk 1]
```diff
@@ -24,10 +24,10 @@
         }
 
         favouriteUrl() {
-            if (this.stock.favourite) {
+            if (this.starHovered) {
+                return starUrls.onHover;
```
Comment (WPFerg) [inline src/sidebars/star/star-controller.js]: Suspected line ending problem here

**Comment by rmarcalo** [comment]
Not sure what you mean by tearout itself, when you click and drag?

**Comment by WPFerg** [comment]
Correct

**Comment by rmarcalo** [comment]
The change of cursor is there to let user know they can interact. When you click and drag I think it should be maintained. This cursor change is also something I would like to test, as it may become too prevalent.

**Pull request merged**